### PR TITLE
Adding aisruss-norwich to master

### LINKED ISSUE #5: aisruss's checklist
- [x] Create an Issue
- [x] Create a Branch 
- [x] Make some Commits
- [x] Create a Pull Request
- [x] Merge the Pull Request
- [x] Dance, Magic Dance!

### PR DESCRIPTION
**Pull request by aisruss**
push problems means i'm behind sorry!
fixes #5

### CODE REVIEW

File: aisruss-norwich.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#Norwich, UK
+
```
Comment (ghost) [inline aisruss-norwich.md:2]: Yumm this all sounds good.

**Review by ghost — COMMENTED** [review]
Thanks for the awesome list! You're the best!

File: aisruss-norwich.md (hunk 1 above)
Comment (GV-JBW) [reply aisruss-norwich.md:2]: I've left a comment.

Comment (aisruss) [reply aisruss-norwich.md:2]: Thanks Eric! Norwich has delicious food!

File: aisruss-norwich.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+##Places to Eat
+- The Glasshouse for :beers: and cheap eats!
+- Olives for brunch
+- The place by St Andrews hall for pizza slices
+
+##Places to Visit
```
Comment (githubteacher) [inline aisruss-norwich.md:8]: @aisruss Do you have a favorite park to visit?

Comment (aisruss) [reply aisruss-norwich.md:8]: Chapelfield has a great park! :)

**Pull request merged**